resources/authgear/templates/en/web/wechat_auth.html

```diff
@@ -25,7 +25,7 @@ <h1 class="font-inherit margin-10 primary-txt">{{ template "wechat-auth-title" }
 >
 {{ template "wechat-open-app" }}
 </a>
-<a class="as-btn btn primary-btn item switch-to-item"
+<a class="as-btn btn primary-btn item switch-to-item wechat-auth-proceed-btn"
```

Comment:
I guess we just need to add the class `wechat` here without introducing `wechat-auth-proceed-btn`.

Reply:
I just afraid in one day if we need update the sso button style, we don't aware we are using it here too. Add comment to the css?

Reply:
Let's add a comment in the wechat CSS block saying that the class name is used in SSO button and the proceed button.